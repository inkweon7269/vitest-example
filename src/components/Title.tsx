const Title = () => {
    return (
        <h1>상품 리스트</h1>
    );
};

export default Title;