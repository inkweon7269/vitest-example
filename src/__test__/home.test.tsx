import {describe, expect, it} from "vitest";
import {render} from "@testing-library/react";
import Title from "@/components/Title";

describe("Title", () => {
    it('toMatchSnapshot', async ()=> {
        const { container } = await render(<Title />)
        expect(container).toMatchSnapshot()
    })
});
