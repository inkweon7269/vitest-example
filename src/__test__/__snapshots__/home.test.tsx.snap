// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Title > toMatchSnapshot 1`] = `
<div>
  <h1>
    상품 리스트
  </h1>
</div>
`;
